src/components/Portfolio.js

```diff
@@ -5,14 +5,26 @@ export default function Portfolio() {
         <div className="portfolio--container">
             <h2 className="portfolio--title">Portfolio</h2>
             <div className="portfolio--link-container">
-                <a href="https://shimmering-malasada-04a43c.netlify.app" target="_blank" rel="noopener noreferrer" className="portfolio--link">
-                    React Notes App
-                </a>
-                <button>
-                    <a href="https://github.com/nngrz/React-project-set/tree/main/notes_app" target="_blank" rel="noopener noreferrer" className="portfolio--btn">
-                        Source Code
+                <div className="projects">
+                    <a href="https://shimmering-malasada-04a43c.netlify.app" target="_blank" rel="noopener noreferrer" className="project--link">
```

Comment:
as you have now two elemenets (two projects), please make it an array, and create a new Component: `<PortfolioEntry link={...} sourceCode={...} />`